### PR TITLE
Updated canton docker image location

### PR DESCRIPTION
**Pull request by daravep**

### CODE REVIEW

**Comment by daravep** [comment]
Note, I updated all locations that use the string digitalasset-canton-enterprise-docker, so should be complete.

File: docs/2.8.0/docs/canton/usermanual/docker.rst [hunk 1]
```diff
@@ -8,15 +8,21 @@
 Running in Docker
 =================
 
+.. note::
+
+	Please note that the images have moved from ``digitalasset-canton-enterprise-docker.jfrog.io/digitalasset/canton-enterprise``
+        to ``digitalasset-docker.jfrog.io/canton-enterprise`` as per version 2.8.0. The old images will remain, but new versions won't 
+        be added to the old repository. In addition, a docker image with the open source version is no longer provided.
```
Comment (carrielaben-da) [inline docs/2.8.0/docs/canton/usermanual/docker.rst:15]:
```suggestion
        be added to the old repository. In addition, a Docker image with the open source version is no longer provided.
```
For clarity, would this be better as "a Docker image is no longer provided with the open source version of Canton"?

File: docs/2.8.0/docs/canton/usermanual/docker.rst [hunk 2]
```diff
@@ -25,61 +31,64 @@ Starting Canton
...
-        --config simple-topology.conf
+   docker run --rm -it digitalasset-docker.jfrog.io/canton-enterprise:latest \
+        --no-tty -C canton.participants.p.ledger-api.port=1234
 
 The ``--rm`` option ensures that the container is removed when the canton process exits.
-The ``-it`` options start the container interactively and provide a TTY for running our console.
-
-The default working directory of the container is ``/canton``.
+The ``-it`` options start the container interactively and provide access to our running console.
+The ``-no-tty`` option disables the ``tty`` mode as otherwise the console is not usable.
```
Comment (carrielaben-da) [inline docs/2.8.0/docs/canton/usermanual/docker.rst:43]:
```suggestion
The ``-no-tty`` option deactivates ``tty`` mode, as the console is not usable with that mode engaged.
```

File: docs/2.8.0/docs/canton/usermanual/docker.rst [hunk 3]
```diff
@@ -25,61 +31,64 @@ Starting Canton
...
 
 Configuring Logging
 -------------------
 The default convention with logging of containers is to have the process to log to ``stdout``. Therefore, we change the logging behaviour of Canton using appropriate :ref:`command line flags <logging>`, such as ``--log-profile=container``.
 
-Supplying custom configuration and DARs
----------------------------------------
+Accessing the Running Node
+--------------------------
+Normally, you will run Canton in ``daemon`` mode and use :ref:`remote console <canton_remote_console>` to interact with the node.
```
Comment (carrielaben-da) [inline docs/2.8.0/docs/canton/usermanual/docker.rst:55]:
```suggestion
Normally, you run Canton in ``daemon`` mode and use :ref:`remote console <canton_remote_console>` to interact with the node.
```

File: docs/2.8.0/docs/canton/usermanual/docker.rst [hunk 4]
```diff
@@ -25,61 +31,64 @@ Starting Canton
...
 
-To expose files to the canton container you must specify a volume mapping from the host machine to the container.
+Exposing the Ledger API or Admin API to the host machine
+--------------------------------------------------------
 
-For example, if you have the local directory ``my-application`` containing your custom canton configuration and DAR:
+Applications using Canton will typically need access to the Ledger or Admin API to read from and write to the ledger.
```
Comment (carrielaben-da) [inline docs/2.8.0/docs/canton/usermanual/docker.rst:60]:
```suggestion
Applications using Canton typically need access to the Ledger or Admin API to read from and write to the ledger.
```

File: docs/2.8.0/docs/canton/usermanual/docker.rst [hunk 5]
```diff
@@ -25,61 +31,64 @@ Starting Canton
...
-      digitalasset-canton-enterprise-docker.jfrog.io/digitalasset/canton-enterprise \
-      --config /canton/my-application/my-config.conf
+      -p 5011:5011 \
+      digitalasset-docker.jfrog.io/canton-enterprise \
+      -C canton.participants.participant1.ledger-api.address=0.0.0.0 \
+      --config examples/01-simple-topology/simple-topology.conf \
+      --bootstrap examples/01-simple-topology/simple-ping.canton
 
-DARs can be loaded using the same container local path.
+The Ledger API port for each participant will need to be mapped separately. The same applies to the Admin API.
```
Comment (carrielaben-da) [inline docs/2.8.0/docs/canton/usermanual/docker.rst:75]:
```suggestion
The Ledger API port for each participant must be mapped separately. The same applies to the Admin API.
```

File: docs/2.8.0/docs/canton/usermanual/docker.rst [hunk 6]
```diff
@@ -25,61 +31,64 @@ Starting Canton
...
 
-Exposing the ledger-api to the host machine
--------------------------------------------
+Supplying custom configuration and DARs
+---------------------------------------
 
-Applications using Canton will typically need access to the ledger-api to read from and write to the ledger.
-Each participant binds the ledger-api to the port specified at the configuration key: ``ledger-api.port``.
-For ``participant1`` in the simple topology example this is set to port 5011.
+To expose files to the canton container, you must specify a volume mapping from the host machine to the container.
```
Comment (carrielaben-da) [inline docs/2.8.0/docs/canton/usermanual/docker.rst:80]:
```suggestion
To expose files to the Canton container, you must specify a volume mapping from the host machine to the container.
```

File: docs/2.8.0/docs/canton/usermanual/docker.rst [hunk 7]
```diff
@@ -25,61 +31,64 @@ Starting Canton
...
 
-To expose the ledger-api to port 5011 on the host machine, run Docker with the following options:
+For example, if you have the local directory ``my-application`` containing your custom canton configuration and DAR:
```
Comment (carrielaben-da) [inline docs/2.8.0/docs/canton/usermanual/docker.rst:82]:
```suggestion
For example, if you have the local directory ``my-application`` containing your custom Canton configuration and DAR:
```

File: docs/2.9.0/docs/canton/usermanual/docker.rst [hunk 8]
```diff
@@ -8,15 +8,21 @@
 Running in Docker
 =================
 
+.. note::
+
+	Please note that the images have moved from ``digitalasset-canton-enterprise-docker.jfrog.io/digitalasset/canton-enterprise``
+        to ``digitalasset-docker.jfrog.io/canton-enterprise`` as per version 2.8.0. The old images will remain, but new versions won't 
+        be added to the old repository. In addition, a docker image with the open source version is no longer provided.
```
Comment (carrielaben-da) [inline docs/2.9.0/docs/canton/usermanual/docker.rst:15]:
```suggestion
        be added to the old repository. In addition, a Docker image with the open source version is no longer provided.
```

File: docs/2.9.0/docs/canton/usermanual/docker.rst [hunk 9]
```diff
@@ -25,61 +31,64 @@ Starting Canton
...
-        --config simple-topology.conf
+   docker run --rm -it digitalasset-docker.jfrog.io/canton-enterprise:latest \
+        --no-tty -C canton.participants.p.ledger-api.port=1234
 
 The ``--rm`` option ensures that the container is removed when the canton process exits.
-The ``-it`` options start the container interactively and provide a TTY for running our console.
-
-The default working directory of the container is ``/canton``.
+The ``-it`` options start the container interactively and provide access to our running console.
+The ``-no-tty`` option disables the ``tty`` mode as otherwise the console is not usable.
```
Comment (carrielaben-da) [inline docs/2.9.0/docs/canton/usermanual/docker.rst:43]:
```suggestion
The ``-no-tty`` option deactivates ``tty`` mode, as the console is not usable with that mode engaged.
```

File: docs/2.9.0/docs/canton/usermanual/docker.rst [hunk 10]
```diff
@@ -25,61 +31,64 @@ Starting Canton
...
 
 Configuring Logging
 -------------------
 The default convention with logging of containers is to have the process to log to ``stdout``. Therefore, we change the logging behaviour of Canton using appropriate :ref:`command line flags <logging>`, such as ``--log-profile=container``.
 
-Supplying custom configuration and DARs
----------------------------------------
+Accessing the Running Node
+--------------------------
+Normally, you will run Canton in ``daemon`` mode and use :ref:`remote console <canton_remote_console>` to interact with the node.
```
Comment (carrielaben-da) [inline docs/2.9.0/docs/canton/usermanual/docker.rst:55]:
```suggestion
Normally, you run Canton in ``daemon`` mode and use :ref:`remote console <canton_remote_console>` to interact with the node.
```

File: docs/2.9.0/docs/canton/usermanual/docker.rst [hunk 11]
```diff
@@ -25,61 +31,64 @@ Starting Canton
...
 
-To expose files to the canton container you must specify a volume mapping from the host machine to the container.
+Exposing the Ledger API or Admin API to the host machine
+--------------------------------------------------------
 
-For example, if you have the local directory ``my-application`` containing your custom canton configuration and DAR:
+Applications using Canton will typically need access to the Ledger or Admin API to read from and write to the ledger.
```
Comment (carrielaben-da) [inline docs/2.9.0/docs/canton/usermanual/docker.rst:60]:
```suggestion
Applications using Canton typically need access to the Ledger or Admin API to read from and write to the ledger.
```

File: docs/2.9.0/docs/canton/usermanual/docker.rst [hunk 12]
```diff
@@ -25,61 +31,64 @@ Starting Canton
...
-      digitalasset-canton-enterprise-docker.jfrog.io/digitalasset/canton-enterprise \
-      --config /canton/my-application/my-config.conf
+      -p 5011:5011 \
+      digitalasset-docker.jfrog.io/canton-enterprise \
+      -C canton.participants.participant1.ledger-api.address=0.0.0.0 \
+      --config examples/01-simple-topology/simple-topology.conf \
+      --bootstrap examples/01-simple-topology/simple-ping.canton
 
-DARs can be loaded using the same container local path.
+The Ledger API port for each participant will need to be mapped separately. The same applies to the Admin API.
```
Comment (carrielaben-da) [inline docs/2.9.0/docs/canton/usermanual/docker.rst:75]:
```suggestion
The Ledger API port for each participant must be mapped separately. The same applies to the Admin API.
```

File: docs/2.9.0/docs/canton/usermanual/docker.rst [hunk 13]
```diff
@@ -25,61 +31,64 @@ Starting Canton
...
 
-Exposing the ledger-api to the host machine
--------------------------------------------
+Supplying custom configuration and DARs
+---------------------------------------
 
-Applications using Canton will typically need access to the ledger-api to read from and write to the ledger.
-Each participant binds the ledger-api to the port specified at the configuration key: ``ledger-api.port``.
-For ``participant1`` in the simple topology example this is set to port 5011.
+To expose files to the canton container, you must specify a volume mapping from the host machine to the container.
```
Comment (carrielaben-da) [inline docs/2.9.0/docs/canton/usermanual/docker.rst:80]:
```suggestion
To expose files to the Canton container, you must specify a volume mapping from the host machine to the container.
```

File: docs/2.9.0/docs/canton/usermanual/docker.rst [hunk 14]
```diff
@@ -25,61 +31,64 @@ Starting Canton
...
 
-To expose the ledger-api to port 5011 on the host machine, run Docker with the following options:
+For example, if you have the local directory ``my-application`` containing your custom canton configuration and DAR:
```
Comment (carrielaben-da) [inline docs/2.9.0/docs/canton/usermanual/docker.rst:82]:
```suggestion
For example, if you have the local directory ``my-application`` containing your custom Canton configuration and DAR:
```

**Review by carrielaben-da — APPROVED** [review]
Generally grammar changes.

File: docs/2.8.0/docs/canton/usermanual/docker.rst (hunk 1 above)
Comment (marciw-da) [reply docs/2.8.0/docs/canton/usermanual/docker.rst:15]: Another option: The open source Canton Docker image is no longer available.

(I'm seeing it this way: the Docker image *is/was* one form of the open source version... not something provided "with" it)

Comment (marciw-da) [reply docs/2.8.0/docs/canton/usermanual/docker.rst:15]: Also wanted to note that we have a separate work item of improving Docker instructions everywhere, so we will likely expand/revise this later (we can go with an "MVP" edition for now) 🙂

Comment (daravep) [reply docs/2.8.0/docs/canton/usermanual/docker.rst]: fixed

File: docs/2.8.0/docs/canton/usermanual/docker.rst (hunk 2 above)
Comment (daravep) [reply docs/2.8.0/docs/canton/usermanual/docker.rst]: Fixed

File: docs/2.8.0/docs/canton/usermanual/docker.rst (hunk 4 above)
Comment (daravep) [reply docs/2.8.0/docs/canton/usermanual/docker.rst]: Fixed

File: docs/2.8.0/docs/canton/usermanual/docker.rst (hunk 5 above)
Comment (daravep) [reply docs/2.8.0/docs/canton/usermanual/docker.rst]: Fixed

File: docs/2.8.0/docs/canton/usermanual/docker.rst (hunk 6 above)
Comment (daravep) [reply docs/2.8.0/docs/canton/usermanual/docker.rst]: Fixed

File: docs/2.8.0/docs/canton/usermanual/docker.rst (hunk 7 above)
Comment (daravep) [reply docs/2.8.0/docs/canton/usermanual/docker.rst]: Fixed

**Pull request merged**